hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -531,7 +530,7 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 
 	// 2. - Reconcile towards expected state of the world.
 	targetConfigVersionHash := hashStruct(config + targetVersion)
-	compressedConfig, err := compress([]byte(config))
+	compressedConfig, err := supportutil.Compress([]byte(config))
```

Comment:
do you think we can consolidate on always using CompressAndEncode/DecodeAndDecompress and drop Compress/Decompress?

Reply:
If we're consistent about it, I don't see why not. I'll make the change but I'll put it in a separate commit for right now (will squash before merging though).